dataactcore/scripts/pull_fpds_data.py

```diff
@@ -1502,7 +1500,7 @@ def get_data(contract_type, award_type, now, sess, sub_tier_list, county_by_name
             except KeyError:
                 continue
 
-            if last_run:
+            if last_run or specific_params:
```

Comment:
I'd like to walk through the different workflows between `last_run`, `specific_params`, and everything else to make sure pairing `last_run` and `specific_params` is fit.